webpack.config.js

```diff
@@ -21,11 +21,11 @@ module.exports = {
     },
     plugins: [
         new webpack.DefinePlugin({
-            TDR_BASE_URL: JSON.stringify(process.env.TDR_BASE_URL || "http://localhost:9000"),
-            TDR_AUTH_URL: JSON.stringify(process.env.TDR_AUTH_URL || "https://tdr.auth.eu-west-2.amazoncognito.com"),
-            TDR_USER_POOL_ID: JSON.stringify(process.env.TDR_USER_POOL_ID || "eu-west-2_6Mn0M2i9C"),
+            TDR_BASE_URL: JSON.stringify(process.env.TDR_BASE_URL || "https://d1nd68699jelkh.cloudfront.net"),
```

Comment:
Will this break local development? The previous defaults meant that you could run `npm run build` in dev, without having to worry about environment variables.